File: Day 2/JavaScript/Level 2/TaskFour.js
const prompt = require('prompt-sync')();

let numberOne = Number(prompt("Enter number: "));
let numberTwo = Number(prompt("Enter number: "));
if (numberOne > numberTwo) console.log("Largest number is", numberOne);
else if (numberOne < numberTwo) console.log("Largest number is", numberTwo);
else console.log("Both numbers are equal");

File: Pre-Ileya/Day 2/Java/Level 2/TaskFour.java
import java.util.Scanner;

public class TaskFour{
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter first number: ");
        double numberOne = scanner.nextDouble();
        System.out.print("Enter second number: ");
        double numberTwo = scanner.nextDouble();
        if (numberOne > numberTwo == 0){
            System.out.printf("%d is the largest number%n", numberOne);
        } 
        else if (numberOne < numberTwo == 0){
            System.out.printf("%d is the largest number%n", numberTwo);
        } else{
            System.out.println("Both numbers are equal");
        }
    }
}
